src/popup/router/components/ClaimTipButton.vue

```diff
@@ -8,22 +8,18 @@
 import Button from './Button';
 import Claim from '../../../icons/claim.svg?vue-component';
 import openUrl from '../../utils/openUrl';
+import { UI_URL } from '../../utils/constants';
 
 export default {
   props: ['styling'],
   components: {
     Button,
     Claim,
   },
-  data() {
-    return {
-      howToClaimURL: 'https://youdonotneedacapetobeahero.com',
-    };
-  },
   created() {},
   methods: {
     openHowToClaimURL() {
-      openUrl(this.howToClaimURL);
+      openUrl(UI_URL);
```

Comment:
Currently this component is not used because `How to claim tips` button was removed in new designs. 
![image](https://user-images.githubusercontent.com/8629465/78749942-ee0b2b80-7977-11ea-94e1-d099b43dd19a.png)